src/constants.js

```diff
@@ -11,6 +11,7 @@ export const DEFAULT_WIDTH = '1680';
 export const DEFAULT_MIN_HEIGHT = '600';
 export const DEFAULT_FILENAME = 'opensearch-report';
 export const DEFAULT_EMAIL_SUBJECT = 'This is an email containing your opensearch dashboard report';
+export const DEFAULT_EMAIL_NOTE = 'Hi,<br>Here is the latest report!';
```

Comment:
will users know to use `<br>` instead of new line?

Reply:
not sure. should we add template is html somewhere in the docs?

Reply:
what happens if new line is provided? if doesn't work as expected maybe programmatically convert text to html?

just a thought but does this need to support reading from files for easier multi-line editing? e.g. `--email-body-file ./email-body.txt`. or users are likely to use one-line notes

Reply:
currently, if new line is provided it just gets ignored 
e.g. 'Hi,\nHere is the latest report!' will look like
<img width="395" alt="image" src="https://user-images.githubusercontent.com/56703525/214389446-15694888-1075-4731-915a-9ae3dcafc7c8.png">

We can surely add text to html logic and/or read from file. Not sure what is preferred though. will check